README.md

```diff
@@ -43,10 +43,10 @@ customerFormValidation
     console.log(validationResult.success); // true
     console.log(validationResult.formGlobalErrors); // []
     console.log(validationResult.fieldErrors);
-    /*[
-        { succeeded: true, type: "REQUIRED", key: "firstName", errorMessage: "" },
-        { succeeded: true, type: "REQUIRED", key: "lastName", errorMessage: "" }
-      ]*/
+    /*{
+        firstName: { succeeded: true, type: "REQUIRED", key: "firstName", errorMessage: "" },
+        lastName: { succeeded: true, type: "REQUIRED", key: "lastName", errorMessage: "" }
+      }*/
```

Comment:
I don't remember what was the intention when we create the readme about this commented code, dou you know why?

Reply:
I see it's dumping the result, it's from a previous version (even previous that current)